tests/popular_repos/update_test_lists.sh

```diff
@@ -0,0 +1,13 @@
+repos=(
+  org_golang_x_crypto
+  org_golang_x_net
+  org_golang_x_sys
+  org_golang_x_text
+  org_golang_x_tools
+  com_github_mattn_go_sqlite3
+  org_golang_google_grpc
+)
+
+for repo in ${repos[*]}; do
```

Comment:
Rather than generating flat lists of tests, then combining them into a build file with Skylark, I'd argue that this script should generate the final BUILD.bazel file. It wouldn't be much extra string formatting, and it would be clearer what's happening. We could drop the `_popular_repos_test_suites` repo rule, too.

Reply:
I would have to write all the exclusion logic, in bash, ug, but you might be right.
Originally I was trying to do this bit in skylark too, but it turned out to be not viable, and I moved the smallest part out that I could.

Reply:
Maybe Python instead?